quickstart/101-key-vault-key/main.tf

```diff
@@ -0,0 +1,65 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = random_pet.rg_name.id
+  location = var.resource_group_location
+}
+
+data "azurerm_client_config" "current" {}
+
+resource "random_string" "azurerm_key_vault_name" {
+  length  = 13
+  lower   = true
+  numeric = false
+  special = false
+  upper   = false
+}
+
+locals {
+  current_user_id = coalesce(var.msi_id, data.azurerm_client_config.current.object_id)
```

Comment:
Looks like this is duplicated in main.tf and variables. tf. please remove one. I believe locals should live in the file where it is used so I'd recommend removing from variables file.

Reply:
Thanks! I meant to move it to main, but copied it instead.